RasterPropMonitor/Core/MonitorPage.cs

```diff
@@ -576,11 +577,12 @@ public void RenderBackground(RenderTexture screen)
                     }
                     break;
                 case BackgroundType.Texture:
+                    //call clear before redraw of textures
                     GL.Clear(true, true, ourMonitor.emptyColorValue);
                     Graphics.DrawTexture(new Rect(0, 0, screen.width, screen.height), backgroundTexture);
                     break;
                 case BackgroundType.Handler:
-
+                    //No clear here as it would interfere with the handlers(Causing effects such as VesselViewer to blink)
```

Comment:
Ahh I see, thanks for explaining that little tweak. I would update mine but I don't have time today.